packages/preferences-controller/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@metamask/preferences-controller",
+  "version": "0.0.0",
+  "description": "Manages user-configurable settings for MetaMask",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/controllers/tree/main/packages/preferences-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/controllers/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/controllers.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "changelog:validate": "../../scripts/validate-changelog.sh",
+    "doc": "../../scripts/doc.sh",
+    "test": "../../scripts/test.sh",
+    "test:watch": "../../scripts/test-watch.sh"
+  },
+  "dependencies": {
+    "@metamask/base-controller": "workspace:~",
+    "@metamask/controller-utils": "workspace:~"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^2.6.0",
+    "@types/jest": "^26.0.22",
+    "deepmerge": "^4.2.2",
+    "jest": "^26.4.2",
+    "jest-it-up": "^2.0.2",
```

Comment:
I would recommend that we avoid using `jest-it-up` until we fix our test race conditions (see: https://github.com/MetaMask/controllers/pull/790#issuecomment-1105858014)

Reply:
Sure, removed here: [8a2819e](https://github.com/MetaMask/controllers/pull/831/commits/8a2819e208976731a58a6e71cd5ad6d4f5dda271)